app/javascript/components/enrollment/steps/Contact.js

```diff
@@ -301,7 +301,7 @@ class Contact extends React.Component {
                 </Form.Group>
                 {this.state.current.patient?.preferred_contact_method?.includes('SMS') &&
                   this.state.current.blocked_sms &&
-                  this.renderWarningBanner('SMS-based reporting selected and this phone number has blocked SMS communications with Sara Alert', true)}
+                  this.renderWarningBanner('SMS-based reporting selected and this phone number has blocked SMS communications with Sara Alert)', true)}
```

Comment:
is this a typo?

```suggestion
                  this.renderWarningBanner('SMS-based reporting selected and this phone number has blocked SMS communications with Sara Alert', true)}
```

Reply:
definitely a typo, thought I fixed this at some point but I guess not.  fixed now: https://github.com/SaraAlert/SaraAlert/pull/1027/commits/7996bd8476129821796c62f818be44c70f49f29a